### ISSUE TITLE
amd64 17.1 profile breaks deepin

### ISSUE THREAD

**Issue by tmu-sprd** [zhtengw/deepin-overlay#60]
There is this news: [Gentoo News: amd64 17.1 profiles are now stable](https://www.gentoo.org/support/news-items/2019-06-05-amd64-17-1-profiles-are-now-stable.html)

Following this, breaks packages from this overlay.

Example: dde-daemon is installed into `/usr/lib64/deepin-daemon`, but dbus service files (and other places) point to `/usr/lib` because hard-coded in source package:
```
$ cat /usr/share/dbus-1/system-services/com.deepin.daemon.Daemon.service
[D-BUS Service]
Name=com.deepin.daemon.Daemon
Exec=/usr/lib/deepin-daemon/dde-system-daemon
User=root
```
source: https://github.com/linuxdeepin/dde-daemon/blob/master/misc/system-services/com.deepin.daemon.Daemon.service

Same with dde-api (https://github.com/linuxdeepin/dde-api/blob/master/misc/system-services/com.deepin.api.Device.service) and most probably all other packages.

Is this already known?

Edit: To be more clear, with profile 17.0 symlinks of `/lib` and `/usr/lib` were pointing to `/lib64` and `/usr/lib64`. With transition to 17.1 following the news, the symlinks point to `/lib32`and `/usr/lib32`. That's why this overlay is working with profile 17.0, but not with 17.1.

**Comment by zhtengw** [comment]
Thank you for the tips. I will read the news and try to fix them.

**Comment by zhtengw** [comment]
I have updated some ebuilds to fix issues with 17.1 profile. If you meet any issues related, please feel free to report it or create a PR.

**Comment by tmu-sprd** [comment]
Did an update and it's working great so far. Closing this as fixed.

Thank you so much for your efforts. Very appreciated.
